Mods/vcmi/config/vcmi/czech.json

```diff
@@ -142,7 +174,16 @@
 	"vcmi.battleWindow.damageEstimation.kills" : "%d zahyne",
 	"vcmi.battleWindow.damageEstimation.kills.1" : "%d zahyne",
 
-	"vcmi.battleResultsWindow.applyResultsLabel" : "Použít výsledek bitvy",
+	"vcmi.battleResultsWindow.applyResultsLabel" : "Použít výsledek bitv"object.core.garrisonVertical.name"y",
```

Comment:
This is invalid JSON

Reply:
Oh thanks, don't know how this happened.